### DISCUSSION TITLE
How to play background music across all the scenes?

### DISCUSSION THREAD

**Discussion by meehjeevan** [axmolengine/axmol#1971] [Q&A]
How to play background music across all the scenes? 
is SimpleAudioEngine not available in Axmol?
I played a sound as background music with AudioEngine::play2d(filename, true);
but it stops when I replaceScene.

**Comment by rh101** [comment]
> is SimpleAudioEngine not available in Axmol?

SimpleAudioEngine was removed in Cocos2d-x v4.

> I played a sound as background music with AudioEngine::play2d(filename, true);
> but it stops when I replaceScene.

Have you tried placing breakpoints on the relevant methods that would stop a sound in AudioEngine, to see where they are being called from when you replace the scene?

**Reply by meehjeevan** [reply]
ok, thanks, I found where it got stopped.

**Reply by paulocoutinhox** [reply]
Hi,

I use:

```
AudioEngine::preload("sounds/music.mp3");
and
AudioEngine::play2d("sounds/music.mp3", true);
```

Sample:
https://github.com/paulocoutinhox/axmol-game-demo/blob/main/Source/AppDelegate.cpp
